common/src/main/java/com/revenuecat/purchases/common/errors.kt

```diff
@@ -42,11 +42,11 @@ fun Exception.toPurchasesError(): PurchasesError {
             PurchasesError(PurchasesErrorCode.InsufficientPermissionsError, localizedMessage)
         }
         else -> PurchasesError(PurchasesErrorCode.UnknownError, localizedMessage)
-    }
+    }.also { errorLog(it.toString()) }
```

Comment:
this is an unusual side-effect to have in a constructor - should this be done wherever the error handling happens instead?

Reply:
I agree. I will move all these logs to wherever these functions are being called from (Backend, BillingWrapper, etc)